matchms/filtering/derive_inchi_from_smiles.py

```diff
@@ -19,6 +23,6 @@ def derive_inchi_from_smiles(spectrum_in: SpectrumType) -> SpectrumType:
             inchi = inchi.rstrip()
             spectrum.set("inchi", inchi)
         else:
-            print("Could not convert smiles", smiles, "to InChI.")
+            logger.warning("Could not convert smiles %s to InChI.", smiles)
```

Comment:
Same here - no info on success, but on failure.

Reply:
True, this also connects to #263.
For now, I can do a first scan and add those missing info-statements for successes at least for the code touched in this PR. We probably might do another round in the future to see where else logging statements would be helpful.

Reply:
Should be mostly done by https://github.com/matchms/matchms/pull/271/commits/cb362654e762ac9c10ff3850057689f74a1d3866 
(time-wise I would suggest to further add logging in another PR)